includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -244,7 +244,9 @@ public function getDiscussLinkData(): array {
 			$tracking = '';
 
 			if ( !empty( $wgEnableDiscussions ) ) {
-				$url = '/d/f';
+				$url = WikiFactory::cityUrlToLanguagePath(
+						WikiFactory::cityIDtoUrl($this->productInstanceId)
```

Comment:
Minor: should be "spacey" per coding conventions, i.e. `WikiFactory::cityIDtoUrl( $this->productInstanceId )`